### DISCUSSION TITLE
Is it possible to setup lazygit to fill entire screen?

### DISCUSSION THREAD

**Discussion by sidlatau** [folke/snacks.nvim#30] [Q&A] [answered]
I see another plugin has a setting `lazygit_floating_window_scaling_factor`(https://github.com/kdheepak/lazygit.nvim?tab=readme-ov-file#usage) to configure how much lazygit should be scaled. Is it possible to do the same with `Snacks`?

**Comment by folke** [comment, marked as answer]
```lua
  {
    "folke/snacks.nvim",
    opts = {
      styles = {
        lazygit = {
          width = 0,
          height = 0,
        },
      },
    },
  }
```

**Reply by folke** [reply]
0 means full width/height.
You can also use fractional values, or integer for absolute

**Reply by xian-wen** [reply]
Hi, when I set it to full width/height, the last line is not shown. How can I solve this?
![image](https://github.com/user-attachments/assets/52706101-d4cc-4433-a1b5-7e8f4bb23fa5)

**Reply by folke** [reply]
This is fixed locally in my dev env, but won't be pushing these changes till I'm back
